src/transformers/models/clvp/number_normalizer.py

```diff
@@ -0,0 +1,238 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""English Normalizer class for Clvp."""
+
+
+import re
+
+
+class EnglishNormalizer:
```

Comment:
Missing `# Copied from`

Reply:
Actually the `EnglishNormalizer` of clvp is quite different from `EnglishNumberNormalizer` of `speecht5` or `whisper`. So # Copied from is not applicable here.